test/ds1621-sync.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+// Test the sychronous raspio-i2c methods
+// Assumes that a DS1621 temperature sensor is wired up to the Pi
+
+var assert = require('assert'),
```

Comment:
I don't regard them as unit tests. At the moment they're there to ensure that raspi-i2c can talk to hardware. I need them for testing at the moment. They can be thrown away later.